dev/ci/scripts/unittests/test_setup.py

```diff
@@ -77,22 +77,22 @@ def test_setup_xml():
     rmtree(RUNDIR)
 
 
-def test_setup_xml_fail_config_env_cornercase():
+def test_setup_workflow_fail_config_env_cornercase():
 
     script_content = ('''#!/usr/bin/env bash
 export HOMEgfs=foobar
-../../../workflow/setup_xml.py "${1}"\n
+../../../workflow/setup_workflow.py "${1}"\n
 ''')
 
-    with open('run_setup_xml.sh', 'w') as file:
+    with open('run_setup_workflow.sh', 'w') as file:
         file.write(script_content)
-    os.chmod('run_setup_xml.sh', 0o755)
+    os.chmod('run_setup_workflow.sh', 0o755)
 
     try:
-        setup_xml_script = Executable(os.path.join(HOMEgfs, "dev/ci/scripts/tests/run_setup_xml.sh"))
-        setup_xml_script.add_default_arg(f"{RUNDIR}/{pslot}")
-        setup_xml_script()
-        assert (setup_xml_script.returncode == 0)
+        setup_workflow_script = Executable(os.path.join(HOMEgfs, "dev/ci/scripts/tests/run_setup_workflow.sh"))
```

Comment:
Maybe instead of hard-coding the `dev/ci/scripts/tests`, the CWD should be captured:

```suggestion
        setup_workflow_script = Executable(os.path.join(os.getcwd(), "run_setup_workflow.sh"))
```